analysis/tests/src/expected/InlayHint.res.txt

```diff
@@ -35,36 +35,12 @@ Inlay Hint src/InlayHint.res 1:34
     "kind": 1,
     "paddingLeft": true,
     "paddingRight": false
-}, {
-    "position": {"line": 14, "character": 17},
-    "label": ": string",
-    "kind": 1,
-    "paddingLeft": true,
-    "paddingRight": false
-}, {
-    "position": {"line": 10, "character": 24},
-    "label": ": int",
-    "kind": 1,
-    "paddingLeft": true,
-    "paddingRight": false
```

Comment:
Why are these gone?

Reply:
I remove inlay hints for parameters. Code Lens already shows the types of the parameters.

Before:
![image](https://user-images.githubusercontent.com/16160544/204798187-59510471-c95d-4421-80e9-025b33281875.png)


After:
![image](https://user-images.githubusercontent.com/16160544/204797929-53481297-bb0c-48c1-b91a-0b4df2c255b0.png)

Reply:
Ah right, you said why already, sorry!